app/src/test/scala/org/alephium/explorer/web/AddressServerSpec.scala

```diff
@@ -154,4 +139,34 @@ class AddressServerSpec()
         }
     }
   }
+
+  "getTransactionsByAddresses" should {
+    "list transactions for an array of addresses" in {
+      forAll(addressGen) { address =>
+        Post("/addresses/transactions", s"""["$address"]""") check { response =>
+          response.as[ArraySeq[Transaction]] is ArraySeq.empty[Transaction]
+        }
+      }
+    }
+
+    "respect the max number of addresses" in {
+      forAll(addressGen)(respectMaxNumberOfAddresses("/addresses/transactions", _))
+    }
+  }
+
+  def respectMaxNumberOfAddresses(endpoint: String, address: Address) = {
```

Comment:
great idea :+1: it's good to add such test for each endpoint having an address limit. I see it as a "regression test", we make sure we don't remove the limit by mistake.